.template/spec/addons/base/github/template_spec.rb

```diff
@@ -67,11 +65,11 @@
     it 'exists' do
       expect(file('.github/wiki/Testing.md')).to exist
     end
+  end
 
-    it 'contains the correct content extracted from README.md' do
-      expect(file('.github/wiki/Testing.md')).to contain('### Docker-based tests on the CI server')
-      expect(file('.github/wiki/Testing.md')).to contain('### Test')
-      expect(file('.github/wiki/Testing.md')).to contain('### Automated Code Review Setup')
+  describe '.github/wiki/Contribution-Guide.md' do
```

Comment:
I think this `describe` block needs to be inside a separate block, just like `describe '.github/wiki/Authentication.md'` or `describe '.github/wiki/CI-CD.md'`